src/algebra/group_power/order.lean

```diff
@@ -320,11 +320,11 @@ by simpa only [sq] using abs_mul_abs_self x
 theorem abs_sq (x : R) : |x ^ 2| = x ^ 2 :=
 by simpa only [sq] using abs_mul_self x
 
-theorem sq_lt_sq (h : |x| < y) : x ^ 2 < y ^ 2 :=
+theorem sq_lt_sq (h : |x| < |y|) : x ^ 2 < y ^ 2 :=
```

Comment:
we should have a nice dot notation for turning `|x| < y` into `|x| < |y|` (maybe `.abs_right`?) otherwise this will quickly get quite annoying to use

Reply:
Maybe, but it's just transitivity with `le_abs_self` so I think it shouldn't be too bad?

Reply:
oh, I got confused by the change in `inner_product_space` so thought it was a bit more annoying. this is fine, then